src/astro/files/base.py

```diff
@@ -38,11 +40,11 @@ def __init__(
         )
 
     @property
-    def path(self) -> str:
+    def path(self) -> Any:
         return self.location.path
 
     @property
-    def conn_id(self) -> str | None:
+    def conn_id(self) -> Any:
```

Comment:
Why do we need to change this?

Reply:
I had not changed this code but getting this mypy error:
```
src/astro/files/base.py: note: In member "conn_id" of class "File":
src/astro/files/base.py:48: error: Returning Any from function declared to
return "Optional[str]"  [no-any-return]
```

Return type is `Optional[str]` which is equivalent to `str | None`

Reply:
The test passed when I reverted the changes @sunank200 in https://github.com/astronomer/astro-sdk/pull/740/commits/96a17d6bb035db24ee99d68bfbb09af9da01816a 

Where was it failing for you?